proofs/logic.lp

```diff
@@ -52,8 +52,55 @@ symbol const eq : ∀a, T a ⇒ T a ⇒ Prop
 symbol const refl  : ∀a x, P (eq a x x)
 symbol const eqind : ∀a x y, P (eq a x y) ⇒ ∀ (p:T a⇒Prop), P (p y) ⇒ P (p x)
 
+// Definition of connectors 
+symbol const bot        : Prop
+symbol const imp        : Prop ⇒ Prop ⇒ Prop
+symbol const {|and|}    : Prop ⇒ Prop ⇒ Prop
```

Comment:
It would be nicer to use unicode symbols for bot, and, or and not, and declare imp, and, or as infix right (associative).

Reply:
For now we can only use unicode for binary symbols (so not `bot`, nor `not`). I'll try to fix that soon.